azext_edge/edge/providers/orchestration/resources/schema_registries.py

```diff
@@ -248,27 +246,29 @@ def delete(
 
     def add_version(
         self,
-        name: str,
+        name: int,
         schema_name: str,
         schema_registry_name: str,
         resource_group_name: str,
-        schema_content: str,
+        schema_version_content: str,
         description: Optional[str] = None,
         current_console: Optional[Console] = None,
     ) -> dict:
-        # TODO: have the schema_content support files too
         from ....util import read_file_content
 
+        if name < 0:
+            raise InvalidArgumentValueError("Version must be a positive number")
```

Comment:
We might be able to put this in a [validator](https://github.com/Azure/azure-iot-ops-cli-extension/blob/dev/azext_edge/edge/_validators.py#L12) so that the command parser validates the arg value before it even gets into this code